packages/esm-patient-medications-app/src/medications-summary/order-basket-action-button.component.tsx

```diff
@@ -26,7 +26,9 @@ const OrderBasketActionButton: React.FC = () => {
         onClick={launchOrdersWorkspace}
       >
         <div className={styles.elementContainer}>
-          <ShoppingCart20 /> {items?.length > 0 && <Tag className={styles.countTag}>{items?.length}</Tag>}
+          <ShoppingCart20 />
+          {workspaces.length > 0 && items?.length > 0 && <Tag className={styles.countTag}>{items?.length}</Tag>}
+          {workspaces.length === 0 && changed === true && <Tag className={styles.countTag}>!</Tag>}
```

Comment:
How would this happen? If all the workspaces are closed, I don't think there can be any pending changes. Right?